Cargo.toml

```
@@ -9,3 +9,9 @@ reqwest = { version = "0.10.0-alpha.2", features = ["json", "blocking"] }
 serde = { version = "1.0.103", features = ["derive"] }
 serde_json = "1.0.44"
 chrono = "0.4.10"
+log = "0.4.8"
+
+[workspace]
+# The executable that uses this library is a member of this workspace so that it
+# can have dependencies github_notifications shouldn't have
+members = ["notification_daemon"]
```

I could add `default-members` here if you want to keep building `notification_daemon` whenever you compile `github_notifications`. `default-members` just defines the packages package-related cargo commands operate on by default:
```suggestion
members = ["notification_daemon"]
default-members = ["notification_daemon"]
```